lua/leadbot/gamemodes/assassins.lua

```diff
@@ -274,13 +274,10 @@ end
 -- fix for timers
 
 hook.Add("Initialize", "LeadBot_TimerFix", function()
-    function DelayNewTarget(ply, delay)
```

Comment:
This removes the override for the gamemode's `DelayNewTarget` function, `ply` might not exist and bots won't acquire new targets.

Reply:
Understood